src/nvector/trilinos/CMakeLists.txt

```diff
@@ -16,22 +16,13 @@
 
 install(CODE "MESSAGE(\"\nInstall NVECTOR_Trilinos\n\")")
 
-# Set Trilinos compilers/flags
-set(CMAKE_CXX_COMPILER ${Trilinos_INTERFACE_CXX_COMPILER})
-set(CMAKE_C_COMPILER ${Trilinos_INTERFACE_C_COMPILER})
-set(CMAKE_CXX_FLAGS
-    "${CMAKE_CXX_FLAGS} ${Trilinos_INTERFACE_CXX_COMPILER_FLAGS}")
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${Trilinos_INTERFACE_C_COMPILER_FLAGS}")
-
 # Create the library
 sundials_add_library(
   sundials_nvectrilinos
   SOURCES nvector_trilinos.cpp
   HEADERS ${SUNDIALS_SOURCE_DIR}/include/nvector/nvector_trilinos.h
   INCLUDE_SUBDIR nvector
-  LINK_LIBRARIES PUBLIC sundials_core
-  OBJECT_LIBRARIES
-  LINK_LIBRARIES PUBLIC SUNDIALS::TRILINOS
+  LINK_LIBRARIES PUBLIC sundials_core Tpetra::all_libs
```

Comment:
What version of Trilinos did `Tpetra::all_libs` become available?

Reply:
v14 which is why the GitHub CI is failing -- it's using v13

Reply:
I'm going to update how we find Trilinos to work like how we find KLU so it should work with older versions before the exported targets were added